src/Persistence.LinnApps/Repositories/StockLocatorRepository.cs

```diff
@@ -27,19 +27,5 @@ public override IQueryable<StockLocator> FilterBy(Expression<Func<StockLocator,
                 .Include(l => l.StorageLocation)
                 .Include(l => l.Part);
         }
-
-        public override async Task<IList<StockLocator>> FilterByAsync(
-            Expression<Func<StockLocator, bool>> filterByExpression,
-            Expression<Func<StockLocator, object>> orderByExpression = null)
-        {
-            var query = this.serviceDbContext.StockLocators.Where(filterByExpression);
-
-            var results = await query
-                              .Include(l => l.StorageLocation)
-                              .Include(l => l.Part)
```

Comment:
I assume these .Includes() are still required so will want to move them over to FilterBy()